docs/source/autoscaling.rst

```diff
@@ -0,0 +1,34 @@
+====================================
+Autoscaling Marathon tasks in PaaSTA
+====================================
+
+PaaSTA allows programmatic control of the number of marathon tasks a service has. It does this by using Zookeeper as a data store to record the number of tasks it thinks should be running for each instance of a service on a per-cluster basis.
+
+How to use autoscaling
+======================
+
+Enabling autoscaling
+--------------------
+
+In order to use autoscaling, edit your ``marathon-*.yaml`` files in your soa configs and add a ``max_instances`` attribute to each instance you want to autoscale. When using autoscaling, the ``instances`` and ``max_instances`` attributes become the minimum and maximum (inclusive) number of marathon tasks paasta will create for your job.
+
+Adding a ``autoscaling_method`` attribute will allow you to specify the logic that determines when to scale up and scale down your instance. If an autoscaling method isn't provided, the ``"default"`` autoscaling method will be used.
+
+Autoscaling methods
+-------------------
+
+The current autoscaling methods are:
+
+:default:
+  The default autoscaling method if none is provided. Tries to use cpu and ram usage to predict when to autoscale. (NOT YET IMPLEMENTED)
+:bespoke:
+  Doesn't autoscale. Allows service authors to implement their own autoscaling.
```

Comment:
"Don't autoscale" isn't exactly right. Just say your second sentence and then clarify that the paasta infrastructure will continue to look into zookeeper for the information, but expects the changes of that data to be initiated from an external source.